models/public/quartznet/quartznet.md

```diff
@@ -0,0 +1,75 @@
+# quartznet
+
+## Use Case and High-Level Description
+
+QuartzNet model is a pair of encoder and decoder. QuartzNet’s design is based on the Jasper architecture,
+which is a convolutional model trained with Connectionist Temporal Classification (CTC) loss.
+The model was trained in NeMo on multiple datasets: LibriSpeech,
+Mozilla Common Voice, WSJ, Fisher, Switchboard, and NSC Singapore English.
+For details see [repository](https://github.com/NVIDIA/NeMo), [paper](https://arxiv.org/pdf/1910.10261.pdf).
+
+## Accuracy
+
+| Metric                       | Value |
+| ---------------------------- | ----- |
+| WER @ Librispeech test-clean | 3.86% |
+
+## Specification
```

Comment:
as I know, we usually define specification, then accuracy.
Could you please swap them?

Reply:
done